eZ/Bundle/EzPublishCoreBundle/Entity/EntityManagerFactory.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\Entity;
+
+use Doctrine\ORM\EntityManagerInterface;
+use eZ\Bundle\EzPublishCoreBundle\ApiLoader\RepositoryConfigurationProvider;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class EntityManagerFactory
```

Comment:
Is this factory internal or meant to be injected in 3rd party?

Reply:
Yes it should be `@internal`. Can't make it final though as it has to be lazy loaded.